captum/optim/_param/image/transforms.py

```diff
@@ -36,6 +47,13 @@ class IgnoreAlpha(nn.Module):
     r"""Ignores a 4th channel"""
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
+        """
+        Ignore the alpha channel.
+        Arguments:
```

Comment:
I think that there is some spacing that we need to do here and for Return.
See: https://github.com/pytorch/captum/blob/master/captum/attr/_core/integrated_gradients.py#L132

Reply:
I've tried to format all the docs to match the formatting used in the integrated_gradients file you linked to.